interface/forms/eye_mag/save.php

```diff
@@ -58,13 +57,12 @@
 require_once("$srcdir/report.inc");
 require_once("$srcdir/html2pdf/html2pdf.class.php");
 
-use Mpdf\Mpdf;
```

Comment:
It's generally best to keep these `use` statement right below the require_once code block so developers can easily surmise what things are included in the script.